### ISSUE TITLE
is this Project dead ?

### ISSUE THREAD

**Issue by Kaabi** [katzgrau/getsparks.org#26]
since codeigniter 3.x is out and sparks is not anymore compatible with ci 3.x, so I'm just wondering if this project dead ?

**Comment by beaufrusetta** [comment]
As far as I know – the project still lives on, and we’ve got great saturation amongst several thousand projects, however, I know that I’ve moved on to Composer integration for my projects instead of Sparks (and I was one of the original guys LOL). Sturgeon has a great (and quick) write up on his blog about it.

Thank you!
Beau

**Comment by kenjis** [comment]
At least, CodeIgniter sparks is not compatible with CodeIgniter 3.0.
But this project is still alive and you can use it with CodeIgniter 2.x.

Right?

**Comment by Kaabi** [comment]
@kenjis yes sparks was very compatible with CI 2.x, but I moved my project to CI 3.x so I did drop sparks and using composer instead.
